fava/static/javascript/main.js

```diff
@@ -91,6 +91,10 @@ e.on('info', (msg) => {
   $('#notifications').insertAdjacentHTML('beforeend', `<li>${msg}</li>`);
 });
 
+e.on('warning', (msg) => {
+  $('#notifications').insertAdjacentHTML('beforeend', `<li class="warning">${msg}</li>`);
```

Comment:
@yagebu How do I add additional args to the `trigger(...)`-call? And how should the auto-dismiss be implemented?